README.rst

```diff
@@ -1472,9 +1471,7 @@ Example usage:
 | default | ``True`` |
 +---------+----------+
 
-If ``True``, attaches markdown rendered HTML of message in email.
-If ``False``, HTML rendering of message will be disabled and a plain
-text email is sent.
+If ``True``, an HTML themed version of the email can be sent using `send_email <#openwisp_utilsadmin_themeemailsend_email>`_ function.
```

Comment:
Let's break this line so it's not too long.

Also, since below we write "the send_email function", I think we should do it here too, therefore it should be: "sent using the send_email function".